.pre-commit-config.yaml

```diff
@@ -28,13 +28,13 @@ repos:
       - id: detect-private-key # detects the presence of private keys.
 
   - repo: https://github.com/pre-commit/pygrep-hooks
-    rev: v1.9.0
+    rev: v1.10.0
     hooks:
       - id: text-unicode-replacement-char
       - id: rst-backticks
 
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.5.1
+    rev: v4.0.0-alpha.8
```

Comment:
Besides this needing to be a seperate PR, I rather see no use of Alpha versions anywhere.

Reply:
fixed and moved to: https://github.com/truecharts/charts/pull/20460